src/test/java/com/artipie/FilesRepoITCase.java

```diff
@@ -154,18 +158,18 @@ void tearDown() {
         this.cntn.stop();
     }
 
-    private String curl(final boolean anonymous, final String action,
-        final ArtipieServer.User usr) throws Exception {
+    private String curl(final String action,
+        final Optional<ArtipieServer.User> usr) throws Exception {
         final String url = "http://host.testcontainers.internal:%d/my-file/file-repo/curl.txt";
         final List<String> cmdlst = new ArrayList<>(
             Arrays.asList(
                 "curl", "-i", "-X", action, String.format(url, this.port)
             )
         );
-        cmdlst.add(this.flag(anonymous));
-        cmdlst.add(this.user(anonymous, usr));
-        String[] cmdarr = new String[cmdlst.size()];
-        cmdarr = cmdlst.toArray(cmdarr);
+        final Pair<String, String> pair = this.flagAndUser(usr);
+        cmdlst.add(pair.getLeft());
+        cmdlst.add(pair.getRight());
```

Comment:
@genryxy when I said about in-lining these methods I meant that we can just put an `if {...}` here and it would be more simple and less verbose:
```java
if (user.isPresent()) {
  cmdlst.add("--user");
  cmdlst.add(String.format(...));
}
```
or even
```java
user.ifPresent(
  usr -> {
    cmdlst.add("--user");
    cmdlst.add(String.format(...));
  }
)
```